src/main/osd/osd.h

```diff
@@ -231,6 +237,9 @@ typedef enum {
     OSD_WARNING_RSSI,
     OSD_WARNING_LINK_QUALITY,
     OSD_WARNING_RSSI_DBM,
+    #ifdef USE_RX_SNR_DBM
```

Comment:
@mikeller does this conditional need to go as well

Reply:
Yes.